docs/internal/admin/Localisation.md

```diff
@@ -36,4 +38,47 @@ Workflow for exporting and importing could be as follows:
 - Export by "File -> Download -> CSV"
 - Re-import using UI!
 
-![Import dialog](/images/google-sheet-import-dialog.png)
\ No newline at end of file
+![Import dialog](/images/google-sheet-import-dialog.png)
+
+## Localisation development
+
+To have access to our context file with all the strings to be localised just import `contexts/Localisation`. Note that you would need to be inside a React component or React hook to be allowed to be using the localisations hook. For fixed enum (defined in the Database) localisation import the `utils/hooks/useLocalisedEnums` file.
+
+### How localisation works?
+
+The app always come with the file `defaultLanguageStrings.js` file which has all the keys of localisation and the base English strings to be used - in case there is a gap on the **main** language to be used (missing key) or even missing the language file... To define other languages to be used is usually done by exporting the existing languages and reimporting with a new language and its strings to be used on the App (as explained before on **Localisation UI (external users)**). In the app the files for localisation are stored in the folder in the server: `./files/localisation` after being imported from one snapshot (`./localisations`).
+
+### Localisations directory
+
+ Contains one languages configuration file `languages.json` and each language folders each containing one file `strings.json` with a map of strings (key) and localised text. The configuration file is easy to follow, and relates to the other language folder in the same directory:
+ ```
+ [
+  {
+    "languageName": "English",
+    "description": "NZ English",
+    "code": "en_nz",
+    "flag": "🇳🇿",
+    "enabled": true
+  },
+  {
+    "languageName": "Portugese",
+    "description": "Portugese translations",
+    "code": "pt_br",
+    "flag": "🇧🇷",
+    "enabled": false
+  },
+  {
+    "languageName": "English",
+    "description": "Certain string replacements for DTAC scholarship",
+    "code": "en_dtac",
+    "flag": "🇦🇺",
+    "enabled": true
+  }
+]
+ ```
+
+ The configuration file bascially has which languages are selectable by the user (`enabled` option), which how it displays to be user for selecion. In case only one language is available, the user don't the 'Language selection' option on the UI. In other cases it will display as the following:
```

Comment:
```suggestion
The configuration file specifies which languages are selectable by the user (`enabled` option), along with how it is displayed to a user in the language selection UI.  If only one language is available, the user doesn't see this option. In other cases it will display like this:
```